system/jlib/jset.hpp

```diff
@@ -138,6 +138,7 @@ interface jlib_decl IBitSet : public IInterface
     virtual void excl(unsigned lo, unsigned hi)     = 0;
     virtual void reset() = 0;
     virtual void serialize(MemoryBuffer &buffer) const = 0;
+    virtual bool matches(const IBitSet& rhs) const = 0;
```

Comment:
I don't think you use this function in your code.  Did you mean to?  If not, then probably best to delete.

Reply:
Otherwise you need to write some unit tests as well as part of the PR

Reply:
Code no longer used by WUDetails so removed.